src/services/apiService.ts

```diff
@@ -74,6 +74,62 @@ export class ApiService {
         return result;
     }
 
+    /**
+     * Returns Tag/Operation pairs matching search request (if specified).
+     * @param searchRequest Search request definition.
+     */
+    public async getOperationsByTags(apiId: string, searchQuery?: SearchQuery): Promise<Page<TagGroup<Operation>>> {
+        const skip = searchQuery && searchQuery.skip || 0;
+        const take = searchQuery && searchQuery.take || Constants.defaultPageSize;
+
+        let query = `apis/${apiId}/operationsByTags?includeNotTaggedOperations=true&$top=${take}&$skip=${skip}`;
+        const odataFilterEntries = [];
+
+        if (searchQuery) {
+            if (searchQuery.tags && searchQuery.tags.length > 0) {
+                const tagFilterEntries = searchQuery.tags.map((tag) => `tag/name eq '${tag}'`);
+                odataFilterEntries.push(`(${tagFilterEntries.join(" or ")})`);
+            }
+
+            if (searchQuery.pattern) {
+                const pattern = Utils.escapeValueForODataFilter(searchQuery.pattern);
+                odataFilterEntries.push(`(contains(operation/name,'${encodeURIComponent(pattern)}'))`);
```

Comment:
should we search by operation displayName?

Reply:
no, this is how operationByTags resource has it right now by some reason.

Reply:
I tried use name, but it doesn't work, displayname works (pics in the description)